src/datastorage/mongo.js

```diff
@@ -11,6 +11,16 @@ userSchema.plugin(passportLocalMongoose);
 
 mongoose.model('User', userSchema);
 
+const albumSchema = new mongoose.Schema({
+  name: String,
+  artist: String,
+  mbid: String,
+  tags: [String],
+  votes: [mongoose.Schema.Types.Mixed],
```

Comment:
Since the vote objects have a predefined structure, we'll want to reflect that in the array of votes property. The `comments` property from this example schema from the mongoose docs illustrates what we're looking for:
```
var mongoose = require('mongoose');
  var Schema = mongoose.Schema;

  var blogSchema = new Schema({
    title:  String,
    author: String,
    body:   String,
    comments: [{ body: String, date: Date }],
    date: { type: Date, default: Date.now },
    hidden: Boolean,
    meta: {
      votes: Number,
      favs:  Number
    }
  });
```